test/auto/injectorSpec.js

```diff
@@ -237,6 +237,15 @@ describe('injector', function() {
     });
 
 
+    // Only Chrome and Firefox support this syntax.
+    if (/chrome|firefox/i.test(navigator.userAgent)) {
+      it('should be possible to annotate functions that are declared using ES6 syntax', function() {
+        // The function is generated using `eval` as just having the ES6 syntax can break some browsers.  
+        expect(annotate(eval('({ fn(x) { return; } })').fn)).toEqual(['x']);
```

Comment:
If I change the test to just
```js
expect(annotate(({ fn(x) { return; } }).fn)).toEqual(['x']);
```

then the JS parser complains about the syntax. It claims there is a missing `:` after `fn` and stops